webui/react/src/styles/antd.scss

```diff
@@ -92,11 +92,11 @@ html {
   /* Select */
 
   .ant-select > .ant-select-arrow {
-    transform: rotate(0deg);
+    transform: translateY(-50%) rotate(0deg);
     transition: transform 0.2s linear;
   }
   .ant-select.ant-select-open > .ant-select-arrow {
-    transform: rotate(180deg);
+    transform: translateY(-50%) rotate(180deg);
   }
```

Comment:
This breaks the arrow in other places
![Screenshot 2021-08-13 at 11-07-49 Experiment 77 - Determined](https://user-images.githubusercontent.com/15078396/129388093-f61052e6-b690-485a-99a2-be16f8148b26.png)

Reply:
weird, I rebased with your changes, and then I took out the arrow changes and it looks fine now...
![Uploading Screen Shot 2021-08-13 at 11.57.09 AM.png…]()

Reply:
Weird, it still looks bad on latest-master, but is fine on netlify